config-v3-documentation

```diff
@@ -55,6 +55,20 @@ data_subdir=tests/
 in_gen=src/gen
 # Source code of generator (without suffix)
 
+in_format=text
+# Format of input:
+#   text    text/plain in ASCII (default)
```

Comment:
The `text/plain` MIME type requires that lines end with CRLF, but písek enforces LF.

Reply:
Yup, that's why is there:
> subject to normalization (newlines, encoding)

Reply:
Sure, but písek doesn't normalize the files to `text/plain`. If we want our input files to be `text/plain` files, písek should replace all LFs by CRLFs before giving them as inputs to solutions or exporting them.